docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md

```diff
@@ -0,0 +1,153 @@
+description: Pantheon private network quickstart on Azure tutorial
+<!--- END of page meta data -->
+
+# Azure Private Network Quickstart tutorial
+
+This tutorial describes how to use the [Pantheon quickstart](https://github.com/PegaSysEng/pantheon-quickstart) 
+to run a private network of Pantheon nodes in a Docker container in a Linux Virtual 
+machine hosted on Microsoft Azure.
+
+## Requirements
+
+* Having an azure account. A [free trial account](https://azure.microsoft.com/en-us/free/) is enough.
+
+## Connect to Azure Portal
+Once you have created an account or if you already have one, login to [portal.azure.com](https://portal.azure.com/)
+
+## Creating resources
+Go on the home page and click on ![Create resource button](create-resource-button-screenshot.png)
+at the top left hand corner.
+
+Search the Marketplace for **Ubuntu Server 18.04 LTS by Canonical** and click on it then click the 
+blue **Create** button at the bottom.
+
+A form will display, fill it with the following values for each step, starting with the **Basics** one:
+
+### Basics step
+  * Subscription --> free trial
+  * Resource group --> create a new one and call it as you like, "pantheon-quickstart" may be a good name.
+  * Virtual machine name --> quickstart
+  * Region --> **Central US** will probably selected by default, don't bother changing it.
+  * Availability options --> No infrastructure redundancy required
+  * Image --> Ubuntu Server 18.04 LTS
+  * Size --> You can keep the proposed hosting plan size as this quickstart won't require a lot 
+  of power and disk space. A **Standard D2s v3 (2 vcpus, 8 GB memory)** will do the job and will be 
+  less expensive.
+  * Authentication type --> choose the one you prefer, as this is not a critical machine, you can just
+  use a password, but a SSH public key is nice too and simpler if you have one.
+  * Username --> provide a user name, here we propose **azure** as a user name and we'll use it all 
+  along this tutorial.
+  * Depending on the authentication method you chose, either fill the password or the public key.
+  * Login with Azure Acrive Directory --> keep it **off**
+  * Public inbound ports --> Allow selected ports
+  * Select inbound ports --> select **HTTP** and **SSH**
+  
+Then go up on the top of the page and switch to the **Guest config** step tab.
+
+### Guest config
+This step aims at installing required software (Docker and Docker-compose) on your virtual machine 
+and retrieve and run the quickstart private network.
+
+To do so, click on the link named **Select an extension to install** and a new resource pane will
+appear on the right.
+
+In the list click on **Custom Script For Linux**, another pane opens with a blue **Create** button at
+its bottom, click on it.
```

Comment:
Getting a failure on this step. A new pane opens, but it's blank with no controls at all. Also does this for other extensions. I've reproed this on Edge and Chrome, on different computers. My repro follows your script except that I selected username/password.

Reply:
and you have windows...